changelog.json

```
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.6.2-beta2": [
+      "[Added] Add manual conflict resolution options in merge conflicts modal - #6062",
+      "[Added] Wire up Syntax Highlighting for Crystal and Julia - #6710. Thanks @KennethSweezy!",
```

Following the pattern from previous releases:

```suggestion
      "[Added] Crystal and Julia syntax highlighting - #6710. Thanks @KennethSweezy!",
```